### ISSUE TITLE
训练自己的版面分析模型在导出inference模型后使用报错

### ISSUE THREAD

**Issue by beetter** [PaddlePaddle/PaddleOCR#11726]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
- 系统环境/System Environment：
ubuntu 22.04
cuda 11.8
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
PaddleDetection-2.6.0
使用picodet_lcnet_x1_0_fgd_layout_cdla训练模型上训练自己的数据集
- 运行指令/Command Code：
export CUDA_VISIBLE_DEVICES=0,1
python -m paddle.distributed.launch --gpus '0,1'  tools/train.py \
    -c configs/picodet/legacy_model/application/layout_analysis/picodet_lcnet_x1_0_layout.yml \
    --eval
- 完整报错/Complete Error Message：
<img width="1016" alt="WechatIMG792" src="https://github.com/PaddlePaddle/PaddleOCR/assets/49197005/27f5c090-7611-4053-ac8d-0ed25e4c0df6">
我用的是configs/picodet/legacy_model/application/layout_analysis/picodet_lcnet_x1_0_layout.yml文件：配置如下：
_BASE_: [
  '../../../../runtime.yml',
  '../../_base_/picodet_esnet.yml',
  '../../_base_/optimizer_100e.yml',
  '../../_base_/picodet_640_reader.yml',
]

pretrain_weights: https://paddledet.bj.bcebos.com/models/pretrained/LCNet_x1_0_pretrained.pdparams
# pretrain_weights: /home/zgc/PaddleDetection-2.6.0/picodet_lcnet_x1_0_fgd_layout_cdla.pdparams
weights: output/picodet_lcnet_x1_0_layout/model_final
find_unused_parameters: True
use_ema: true
cycle_epoch: 10
snapshot_epoch: 1
epoch: 100

PicoDet:
  backbone: LCNet
  neck: CSPPAN
  head: PicoHead

LCNet:
  scale: 1.0
  feature_maps: [3, 4, 5]

metric: COCO
num_classes: 15

TrainDataset:
  !COCODataSet
    image_dir: train
    anno_path: annotations/instance_train.json
    dataset_dir: ./dataset/publaynet/cocome311/
    data_fields: ['image', 'gt_bbox', 'gt_class', 'is_crowd']

EvalDataset:
  !COCODataSet
    image_dir: val
    anno_path: annotations/instance_val.json
    dataset_dir: ./dataset/publaynet/cocome311/

TestDataset:
  !ImageFolder
    anno_path: ./dataset/publaynet/cocome311/annotations/instance_val.json

worker_num: 8
eval_height: &eval_height 800
eval_width: &eval_width 608
eval_size: &eval_size [*eval_height, *eval_width]

TrainReader:
  sample_transforms:
  - Decode: {}
  - RandomCrop: {}
  - RandomFlip: {prob: 0.5}
  - RandomDistort: {}
  batch_transforms:
  - BatchRandomResize: {target_size: [[768, 576], [800, 608], [832, 640]], random_size: True, random_interp: True, keep_ratio: False}
  - NormalizeImage: {is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
  - Permute: {}
  batch_size: 24
  shuffle: true
  drop_last: true
  collate_batch: false

EvalReader:
  sample_transforms:
  - Decode: {}
  - Resize: {interp: 2, target_size: [800, 608], keep_ratio: False}
  - NormalizeImage: {is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
  - Permute: {}
  batch_transforms:
  - PadBatch: {pad_to_stride: 32}
  batch_size: 24
  shuffle: false

TestReader:
  inputs_def:
    image_shape: [1, 3, 800, 608]
  sample_transforms:
  - Decode: {}
  - Resize: {interp: 2, target_size: [800, 608], keep_ratio: False}
  - NormalizeImage: {is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
  - Permute: {}
  batch_transforms:
  - PadBatch: {pad_to_stride: 32}
  batch_size: 24
  shuffle: false
导出命令为：
python  tools/export_model.py \
-c configs/picodet/legacy_model/application/layout_analysis/picodet_lcnet_x1_0_layout.yml \
-o export.benchmark=True weights='output/picodet_lcnet_x1_0_layout/best_model.pdparams' \
--output_dir=output_inference
当export.benchmark=True时，导出的模型文件有3个和1个yml文件，但是可以使用效果很差，当export.benchmark=False也是有4个文件，程序就会出现ValueError: (InvalidArgument) The conv2d_fusion Op's Input Variable `Input` contains uninitialized Tensor.
  [Hint: Expected t->IsInitialized() == true, but received t->IsInitialized():0 != true:1.] (at /paddle/paddle/fluid/framework/operator.cc:2411)
  [operator < conv2d_fusion > error]。请问遇到这个情况怎么解决。
请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by TingquanGao** [comment]
建议使用Paddle2.5.2版本尝试

**Comment by TingquanGao** [comment]
建议都使用2.5.2

**Comment by TingquanGao** [comment]
长时间未回复，已close，如有问题，可reopen并补充
